configs/platforms/fedora-f26-x86_64.rb

```diff
@@ -0,0 +1,10 @@
+platform "fedora-f26-x86_64" do |plat|
+  plat.servicedir "/usr/lib/systemd/system"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "systemd"
+
+  plat.provision_with "/usr/bin/dnf install -y --best --allowerasing autoconf automake rsync gcc make rpmdevtools rpm-libs"
```

Comment:
@McdonaldSeanp you already noticed this while we were talking earlier, but for posterity: gcc won't need to be installed here

Reply:
This uncovered a separate issue, so gcc will need to be taken out in a new PR: See https://tickets.puppetlabs.com/browse/PA-1487 for details